package.json

```
@@ -16,17 +16,18 @@
     "prepare": "git config --local core.hooksPath .githooks",
     "prettier": "prettier --write \"./**/*.{js,json,md,ts,yml}\"",
     "sort-package-json": "sort-package-json \"package.json\" \"packages/*/package.json\"",
-    "install-browser-deps": "npm install webdriverio@8.13.12 playwright@1.36.1 @vitest/browser@0.32.2"
+    "install-browser-deps": "npm install webdriverio @vitest/browser web-streams-polyfill"
   },
   "devDependencies": {
     "@types/estree": "^1.0.1",
     "@types/node": "18.11.9",
     "@types/tape": "4.13.2",
     "@typescript-eslint/eslint-plugin": "5.33.1",
     "@typescript-eslint/parser": "5.33.1",
-    "@vitest/coverage-v8": "0.32.2",
-    "@vitest/ui": "0.32.2",
-    "c8": "7.12.0",
+    "@vitest/browser": "^1.2.0",
```

Should this be the caret version of 1.2.0 or `1.2.0`? 
```suggestion
    "@vitest/browser": "1.2.0",
```